datalake/backend/processing/processing_assets_model.py

```diff
@@ -0,0 +1,12 @@
+from pynamodb.attributes import UnicodeAttribute
+from pynamodb.models import Model
+
+
+class ProcessingAssetsModel(Model):
+    class Meta:  # pylint:disable=too-few-public-methods
+        table_name = "processing_assets"
+        region = "ap-southeast-2"  # TODO: don't hardcode region
+
+    pk = UnicodeAttribute(hash_key=True)
```

Comment:
@l0b0 , looks like you are missing `sk` field in your model

Reply:
I stopped working on it because I think we should probably just have a single table for the whole application. That's the standard model for DynamoDB, confirmed in #cop-aws.